File: e2e/testdata/mock-repo-multilang/src/Main.java
public class Main {
    public static void main(String[] args) {
        // Java environment variable usage
        String apiKey = System.getenv("API_KEY");
        String dbUrl = System.getenv().get("DATABASE_URL");
        String secret = System.getenv("SECRET_KEY");
        String port = System.getenv("PORT");
        
        System.out.println(apiKey + " " + dbUrl + " " + secret + " " + port);
    }
}

File: e2e/testdata/mock-repo-multilang/src/main.rs
use std::env;

fn main() {
    // Rust environment variable usage
    let api_key = env::var("API_KEY").unwrap_or_default();
    let db_url = std::env::var("DATABASE_URL").unwrap_or_default();
    let secret = env::var_os("SECRET_KEY");
    let port = std::env::var_os("PORT");
    
    println!("{} {} {:?} {:?}", api_key, db_url, secret, port);
}

